packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -22,37 +23,50 @@ import {
   OutboundShipmentSummaryItem,
   OutboundShipmentRow,
 } from './types';
+import { getDataSorter } from '../utils';
+
+const getExistingLine = (
```

Comment:
hmm this is where I thought maybe this idea sucked.

Basically searching for an existing batch is a bit annoying 🤷